third-party/install-gcc.sh

```diff
@@ -1,115 +0,0 @@
-#! /usr/bin/env bash
```

Comment:
Why delete this?

Reply:
now use system gcc or user install itselt

Reply:
User of course could try gcc him want. And this script is used by our infra now, plz check it first instead of delete directly.

Reply:
It's better not to delete the script until we've finished the discussion about compiler usage.

Reply:
ok.